src/NzbDrone.Automation.Test/Sonarr.Automation.Test.csproj

```diff
@@ -5,6 +5,10 @@
   <ItemGroup>
     <PackageReference Include="Selenium.Support" Version="3.141.0" />
     <PackageReference Include="Selenium.WebDriver.ChromeDriver" Version="92.0.4515.10700" />
+    <PackageReference Update="NUnit" Version="3.13.3" />
```

Comment:
Hmm, where these packages used implicitly? Rider for some reason decided to specify them explicitly